### ISSUE TITLE
IF/THEN line ending in ELSE causes problems with the auto-indent feature

### ISSUE THREAD

**Issue by CHR-IS-B** [rubberduck-vba/Rubberduck#2233]
A line of code like this:
`If 1 > 0 Then MsgBox "True!" Else`
(which is poor but valid VBA) causes the indent feature to indent incorrectly for all lines below it.

When I ran the indent feature on this:

```
Public Sub Delete_Hidden_Rows_Columns()
    For lp = 256 To 1 Step -1                    'loop through all columns
        If ws.Columns(lp).EntireColumn.Hidden = True Then ws.Columns(lp).EntireColumn.Delete Else
    Next

    For lp = 65536 To 1 Step -1                  'loop through all rows
        If ws.Rows(lp).EntireRow.Hidden = True Then ws.Rows(lp).EntireRow.Delete
    Next lp
End Sub
```

I got this:

```
Public Sub Delete_Hidden_Rows_Columns()
    For lp = 256 To 1 Step -1                    'loop through all columns
        If ws.Columns(lp).EntireColumn.Hidden = True Then ws.Columns(lp).EntireColumn.Delete Else
        Next

        For lp = 65536 To 1 Step -1              'loop through all rows
            If ws.Rows(lp).EntireRow.Hidden = True Then ws.Rows(lp).EntireRow.Delete
        Next lp
    End Sub
```

Looks like it's compounded if there are multiple one-line IF/THEN statements ending in ELSE.

**Comment by retailcoder** [comment]
@comintern is this a known issue?

**Comment by comintern** [comment]
@retailcoder - It is now...  :-)  Should be a super quick fix (and a couple new unit test) though.

**Comment by comintern** [comment]
This was actually broken during the fix to #1858 due to a test case that actually wasn't compilable code.  Fixed in my next PR.
